db/migrate/20211102100659_add_digest_extracted_at_to_settings.rb

```diff
@@ -0,0 +1,5 @@
+class AddDigestExtractedAtToSettings < ActiveRecord::Migration[6.1]
```

Comment:
Is this migration already in https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/3011? (not sure if this will cause any issues)